extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -60,40 +67,39 @@ public function execute() {
 		$pageId = $service->getCanonicalPageIdFromPageId( $this->currentPageId );
 
 		// we still assume the response is the same format as MediaWiki's
-		$response   = $service->getParseResponseFromPageId( $pageId );
+		$response = $service->getParseResponseFromPageId( $pageId );
 
 		// ensure the response is an array, even if empty.
-		$response   = $response == false ? array() : $response;
-		$titleStr   = $service->getTitleStringFromPageId( $pageId );
+		$response = $response == false ? [] : $response;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)